docs/sandbox/SiriAzureUpdaters.md

```diff
@@ -0,0 +1,7 @@
+# Siri Azure Updater
```

Comment:
This file should be removed, as there is now [docs/sandbox/SiriAzureUpdater.md](https://github.com/opentripplanner/OpenTripPlanner/pull/4106/files#diff-84fc603c8b925b8c740f7d9f0d4fdc8b807c7d2c60a2ddff2f422ccdbe211b89)